app/forms/claims/mentor_form.rb

```diff
@@ -1,10 +1,18 @@
 class Claims::MentorForm < ApplicationForm
-  FORM_PARAMS = %i[trn].freeze
+  include ActiveModel::Attributes
+  FORM_PARAMS = [:trn, "date_of_birth(1i)", "date_of_birth(2i)", "date_of_birth(3i)"].freeze
 
-  attr_accessor :school, :trn, :first_name, :last_name
+  attribute :school
+  attribute :trn
+  attribute :first_name
+  attribute :last_name
+  attribute "date_of_birth(1i)"
+  attribute "date_of_birth(2i)"
+  attribute "date_of_birth(3i)"
```

Comment:
Could these be `attr_accessor`s instead? `ActiveModel::Attributes` adds type-safety/casting.